website/pages/docs/from-1.5/onlyexcept.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: Only Except - HCL Configuration Language
+sidebar_title: Only Except
+description: >-
+  Only and Except can be used as a command line argument to selectively run
+  builds. Only and Except can also be used in a provisioner to not run it for a
+  source.
+---
+
+# Only and Except
```

Comment:
I really like this putting all together cli and template only/except. Less chance to have confusions around these. :D